test-stress-options.scm does not exist, and test-qof passes when run in the
build directory after the gnucash build.

diff -ur gnucash-3.3.old/gnucash/report/standard-reports/test/CMakeLists.txt gnucash-3.3/gnucash/report/standard-reports/test/CMakeLists.txt
--- gnucash-3.3.old/gnucash/report/standard-reports/test/CMakeLists.txt	2018-10-04 09:29:00.916641417 -0400
+++ gnucash-3.3/gnucash/report/standard-reports/test/CMakeLists.txt	2018-10-04 09:30:52.962504860 -0400
@@ -13,10 +13,6 @@
   test-income-gst.scm
 )
 
-set(scm_test_with_textual_ports_SOURCES
-    test-stress-options.scm
-)
-
 set(GUILE_DEPENDS
   scm-gnc-module
   scm-app-utils
@@ -31,9 +27,6 @@
 
 if (HAVE_SRFI64)
   gnc_add_scheme_tests("${scm_test_with_srfi64_SOURCES}")
-  if (HAVE_TEXT_PORTS)
-    gnc_add_scheme_tests("${scm_test_with_textual_ports_SOURCES}")
-  endif (HAVE_TEXT_PORTS)
 endif (HAVE_SRFI64)
 
 gnc_add_scheme_tests("${scm_test_standard_reports_SOURCES}")
diff -ur gnucash-3.3.old/libgnucash/engine/test/CMakeLists.txt gnucash-3.3/libgnucash/engine/test/CMakeLists.txt
--- gnucash-3.3.old/libgnucash/engine/test/CMakeLists.txt	2018-10-04 09:29:00.876640751 -0400
+++ gnucash-3.3/libgnucash/engine/test/CMakeLists.txt	2018-10-05 10:46:22.542962546 -0400
@@ -54,8 +54,6 @@
 # This test does not run on Win32
 if (NOT WIN32)
   set(SOURCES ${test_qof_SOURCES} ${CMAKE_SOURCE_DIR}/common/test-core/unittest-support.c)
-  add_engine_test(test-qof "${SOURCES}")
-  target_compile_definitions(test-qof PRIVATE TESTPROG=test_qof)
 
   set(SOURCES ${test_engine_SOURCES} ${CMAKE_SOURCE_DIR}/common/test-core/unittest-support.c)
   add_engine_test(test-engine "${SOURCES}")
